guides/common/modules/proc_listing-available-scap-contents.adoc

```diff
@@ -0,0 +1,22 @@
+[id="listing-available-scap-contents_{context}"]
+= Listing Available SCAP Contents
+
+Use this procedure to view what SCAP contents are already loaded in {Project}.
+To use the CLI instead of the {ProjectWebUI}, see the xref:cli-listing-available-scap-contents_{context}[CLI procedure].
+
+.Prerequisite
+* Your user account has the `view_scap_contents` permission.
+
+.Procedure
+* In the {ProjectWebUI}, navigate to *Hosts* > *SCAP contents*.
+
+[id="cli-listing-available-scap-contents_{context}"]
+.CLI procedure
+* Run the following Hammer command on {ProjectServer}:
```

Comment:
I assume you did not change this (so it also affects "master"), but in general, I like something like "List available SCAP content on your {ProjectServer}" better than "run the following command ...". This does not block your PR.

Reply:
I agree, but let's do this from master.